packages/@ourworldindata/grapher/src/schema/grapher-schema.003.yaml

```diff
@@ -200,7 +200,7 @@ properties:
         type: string
         description: Url of the concrete schema version to use to validate this document
         format: uri
-        default: "https://files.ourworldindata.org/schemas/grapher-schema.003.json"
+        default: "https://files.ourworldindata.org/schemas/grapher-schema.latest.json"
```

Comment:
No here we want to always point to a specific version. When you load a schema from https://files.ourworldindata.org/schemas/grapher-schema.latest.json then you should be able to understand from the contents of the file which precise schema version this is. If we would use latest here you couldn't.

Another way to think about it is that latest is just a convenient shortcut. In all places where the URL might be persisted we should make sure to use a precise version so that we'll know that the right schema version will be used.

Does that make sense? This stuff is a bit weird to wrap your head around, happy to chat if that that helps resolve any remaining questions :)

Reply:
But we should mention in the readme that this version number should be increased if you increase the one on the top :)

Reply:
But we would also need to increase the version number for all existing configs, right? Because the codebase only knows how to handle configs that comply with the latest schema. Does this mean that whenever I introduce a breaking change to the schema and write a database migration to account for it, I need to increase this version number as well for all configs that need to be updated?